src/content/blog/2025-01-22-eslint-2024-year-review.md

```diff
@@ -0,0 +1,156 @@
+
+### Expenses review
+
+![2024 Expenses - Summarized in following table](/assets/images/blog/2025/2024-expenses.svg)
+
+In 2024, we spent $247,669.54 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. This is in comparison to the $187,964.30 of income for a **net loss of -$59,705.24**. See the table below for an overview.
+
+
+| **Category**                | **2024 Amount**  | **2023 Amount** | **Gain/Loss %** |
+|-----------------------------|------------------|-----------------|-----------------|
+| Maintenance and Development | $147,273.43      | $66,910.37      | +120.12%        |
+| TSC Stipend (Tidelift)      | $18,449.16       | $30,077.84      | -38.68%         |
+| Triage and Support          | $3,985.50        | $10,307.78      | -61.33%         |
+| Technical Writing           | $0.00            | $8,155.00       | -100.00%        |
+| Contributor Pool            | $14,100.30       | $4,500.00       | +213.34%        |
+| Payment Processor Fees      | $3,808.97        | $2,854.51       | +33.45%         |
+| Health Insurance            | $12,698.40       | $13,447.41      | -5.57%          |
+| Open Source Collective Fee  | $11,696.69       | $9,206.88       | +27.04%         |
+| Ecosystem/Dependencies      | $16,100.00       | $24,400.00      | -33.99%         |
+| Community Management        | $9,825.03        | $15,599.84      | -37.04%         |
+| Google Workspace            | $182.06          | $187.35         | -2.82%          |
+| Code Explorer               | $7,200.00        | $0.00           | +100%           |
+| Config Inspector            | $2,350.00        | $0.00           | +100%           |
+| **Total**                   | **$247,669.54**  | **$185,646.98** | **+33.39%**     |
+
+
+2024 was a record year for ESLint in many ways. The amount of work completed was due to a large number of contributions not only from the team but also from outside contributors. Our maintenance and development costs soared this year as a result, as did our Contributor Pool costs, which is what we pay outside contributors for significant contributions to ESLint. We had additional help triaging issues and PRs, which also contributed to the increase in costs.
+
+How were we able to make up the difference between income and expenses? The ESLint team has long had a belief in building up at least a year of reserves so that any abrupt disruption of income wouldn't disrupt the project immediately. Prior to 2023, we were mostly focusing on bug fixes, stability, and keeping up with changes to the JavaScript language. During that time, costs were relatively low and so we spent less than we received in order to build up our reserves. Starting in 2023, we started significantly overhauling the ESLint project, and a result, that was the first year we spent slightly more than we took in. 2024 accelerated those changes as more contributions led to higher costs.
```

Comment:
```suggestion
How were we able to make up the difference between income and expenses? The ESLint team has long had a belief in building up at least a year of reserves so that any abrupt disruption of income wouldn't disrupt the project immediately. Prior to 2023, we were mostly focusing on bug fixes, stability, and keeping up with changes to the JavaScript language. During that time, costs were relatively low so we spent less than we received in order to build up our reserves. Starting in 2023, we started significantly overhauling the ESLint project, and as a result, that was the first year we spent slightly more than we took in. 2024 accelerated those changes as more contributions led to higher costs.
```